tools/visualModelEditor/src/mermaid-editor-utils.ts

```diff
@@ -75,9 +75,14 @@ export function getType(typeDef) {
 }
 
 export function getModel(schema: NormalizedEdmModel) {
+  var entityContainer = schema["$EntityContainer"];
+  if (!entityContainer) {
+    return null;
+  }
+
   var modelName;
   var serviceName;
-  [modelName, serviceName] = schema["$EntityContainer"].split(".");
+  [modelName, serviceName] = entityContainer.split(".");
```

Comment:
This only works for model names without dots and will fail for `"$EntityContainer": "microsoft.graph.GraphService"`

Reply:
Made more robust, here and elsewhere, in latest commit.